fdbrpc/Stats.actor.cpp

```diff
@@ -99,6 +99,8 @@ ACTOR Future<Void> traceCounters(std::string traceEventName,
 	for (ICounter* c : counters->counters)
 		c->resetInterval();
 
+	state Reference<EventCacheHolder> tempEventHolder = makeReference<EventCacheHolder>(trackLatestName);
```

Comment:
In this function, we allow callers to pass in an empty string to signify that they don't want to activate the track latest feature. If the string is empty, we can just leave this as a default constructed reference and then nothing will happen.

Also a minor nit: I would remove the `temp` prefix of the name here because it suggests that this holder is in some way temporary.